app/metrics/support/counter_support.rb

```diff
@@ -0,0 +1,21 @@
+module CounterSupport
+  def run
```

Comment:
I wonder whether `run` is the right name: it seems to imply some kind of process, whereas this is fetching and formatting the current data.